mlrun/db/httpdb.py

```diff
@@ -2770,6 +2770,74 @@ def trigger_migrations(self) -> Optional[schemas.BackgroundTask]:
             )
         return None
 
+    def dataschema_to_list(self):
+        """Provide data schema (feature store schemas) of MLRun projects as collection.
+
+        Examples::
+
+            import mlrun.db.httpdb as mldb
+            import json
+
+            # connection to solution
+            db = mldb.HTTPRunDB('http://localhost:8080')
+            # or db = mldb.HTTPRunDB()
+
+            # print data schema of all projects in JSON format
+            print(json.dumps(db.dataschema_to_list(),
+                ensure_ascii = False,
+                indent=2,
+                allow_nan=True)
+            )
+
+        :return:    Collection of data schema (feature store schemas)
+        """
+
+        dataschema = []
+
+        # iteration via projects
+        for prj in self.list_projects():
+            dataschema.append({})
+            dataschema[-1]["project_name"] = prj.name
+            if prj.spec.description:
+                dataschema[-1]["project_description"] = prj.spec.description
+
+            # iteration via feature sets
+            fsets = self.list_feature_sets(prj.name)
+            if fsets:
+                featureset = []
+                dataschema[-1]["featureset"] = featureset
+                for fset in fsets:
+                    if fset:
+                        featureset.append({})
+                        featureset[-1]["name"] = fset.fullname
+                        if fset.spec.description:
+                            featureset[-1]["description"] = fset.spec.description
+
+                        # iteration via entities
+                        if fset.spec.entities:
+                            entities = []
```

Comment:
This entire iteration can also be replaced by something like:
```
entities = fset.spec.entities.to_dict()
```
Or you can also pass to `to_dict()` a set of fields to retrieve, if you're not interested in all the fields:
```
entities = [entity.to_dict(['name', 'description', 'value_type']) for entity in fset.spec.entities]
```
Same is of course true for the features.